kubert/src/log.rs

```diff
@@ -4,10 +4,34 @@ use thiserror::Error;
 
 pub use tracing_subscriber::{util::TryInitError as LogInitError, EnvFilter as LogFilter};
 
+#[cfg(feature = "clap")]
+use clap::{Arg, ArgEnum, ArgMatches, Args, Command, FromArgMatches};
+
+/// Configures logging settings.
+#[derive(Debug)]
+#[cfg_attr(docsrs, doc(cfg(feature = "log")))]
+pub struct LogArgs {
+    /// The log format to use.
+    pub log_format: LogFormat,
+
+    /// The filter that determines what tracing spans and events are enabled.
+    pub log_level: LogFilter,
```

Comment:
Unfortunately, using this eliminates the ability to configure logging via the environment.

Reply:
I think the same issues exist with the log format (with regard to env variables, etc).

Is there any coupling between the tokio-console setting and the other log configuration?

Reply:
> I think the same issues exist with the log format (with regard to env variables, etc).
> 
> Is there any coupling between the tokio-console setting and the other log configuration?

No, it's just also a required argument to the function for actually constructing the `tracing` subscriber, and I thought it was nice to generate a CLI argument for it as well. We could just as well add a `LogBuilder` struct or something instead of `LogArgs`, so that the user can define the CLI args themselves.

Reply:
...I just had a crazy thought for how we could do the env vars actually

Reply:
I think we'll want to add our own macros/main at some point in the future, but for now i'd be inclined to add this new flag as a one-off.

Is the tokio console server configurable at all? especially what port it runs on?